scripts/setup.sh

```diff
@@ -225,13 +225,20 @@ if [ $no_dependencies -eq 0 ]; then
     sudo="/usr/bin/sudo"
     check_if_exists "$sudo"
 
+    # debian based build_requirements
+    if [ \( "$os" == "ubuntu" -o "$os" == "debian" \) ]; then
+        build_requirements='python-dev build-essential libffi-dev libssl-dev'
+	else
+		build_requirements='' # fill me in for other platforms
```

Comment:
I ran through this on CentOS just now, and the deps required are ``python-devel libffi-devel openssl-devel gcc``. Can you add an ``if []`` here for those?